services/gateway/apis/http-version2/swagger/definitions/accounts.json

```diff
@@ -0,0 +1,393 @@
+                                "address": {
+                                    "type": "string",
+                                    "format": "address",
+                                    "example": "lskdwsyfmcko6mcd357446yatromr9vzgu7eb8y99",
+                                    "description": "The Lisk Address is the human readable representation of the accounts owners' public key.\nIt is 41 character long address thats begins with `lsk`.\n"
+                                },
+                                "publicKey": {
+                                    "type": "string",
+                                    "format": "publicKey",
+                                    "example": "968ba2fa993ea9dc27ed740da0daf49eddd740dbd7cb1cb4fc5db3a20baf341b",
+                                    "description": "The public key is derived from the private key of the owner of the account.\nIt can be used to validate that the private key belongs to the owner, but not provide access to the owners private key.\n"
+                                },
+                                "isMandatory": {
+                                    "type": "boolean",
+                                    "example": "false"
+                                }
+                            }
+                        }
+                    },
+                    "memberships": {
+                        "type": "array",
+                        "items": {
+                            "type": "object",
+                            "properties": {
+                                "address": {
+                                    "type": "string",
+                                    "format": "address",
+                                    "example": "lskdwsyfmcko6mcd357446yatromr9vzgu7eb8y99",
+                                    "description": "The Lisk Address is the human readable representation of the accounts owners' public key.\nIt is 41 character long address thats begins with `lsk`.\n"
+                                },
+                                "publicKey": {
+                                    "type": "string",
+                                    "format": "publicKey",
+                                    "example": "968ba2fa993ea9dc27ed740da0daf49eddd740dbd7cb1cb4fc5db3a20baf341b",
+                                    "description": "The public key is derived from the private key of the owner of the account.\nIt can be used to validate that the private key belongs to the owner, but not provide access to the owners private key.\n"
+                                }
+                            }
+                        }
+                    }
+                }
+            },
+            "dpos": {
+                "type": "object",
+                "properties": {
+                    "delegate": {
+                        "$ref": "#/definitions/delegate"
+                    }
+                }
+            }
```

Comment:
Dpos is missing `sendVotes` and `unlocking`
```
      "sentVotes": [
        {
          "delegateAddress": "lsk24cd35u4jdq8szo3pnsqe5dsxwrnazyqqqg5eu",
          "amount": "102000000000"
        },
        {
          "delegateAddress": "lsk24cd35u4jdq8szo3pnsqe5dsxwrnazyqqqg5eu",
          "amount": "95000000000"
        }
      ],
      "unlocking": [
        {
          "delegateAddress": "lsk24cd35u4jdq8szo3pnsqe5dsxwrnazyqqqg5eu",
          "amount": "string number",
          "unvoteHeight": 30
        }
```

Reply:
Done!